index.js

```diff
@@ -34,8 +34,11 @@ breadBot.on('ready', () => {
 const TIME_BEFORE_REACTION_COUNT = process.env.timebeforereactioncount * 1000 * 60;
 const leaderboardSlot = process.env.amountonleaderboard
 
-function createLeaderBoard(serverScores, message) {
-	let leaderboardSpots = leaderboardSlot;
+
+
+function createLeaderBoard(serverScores, message, slots = leaderboardSlot) {
+	slots = slots > 20 ? slots = 20 : slots;
+	let leaderboardSpots = slots;
```

Comment:
The heck is this mess, just use slots as a variable for everything? No reason for leaderboardSpots to exist?